pkg/router/contour.go

```diff
@@ -23,10 +23,13 @@ type ContourRouter struct {
 	contourClient clientset.Interface
 	flaggerClient clientset.Interface
 	logger        *zap.SugaredLogger
+	ingressClass  string
 }
 
 // Reconcile creates or updates the HTTP proxy
 func (cr *ContourRouter) Reconcile(canary *flaggerv1.Canary) error {
+	annotation := "projectcontour.io/ingress.class"
```

Comment:
nit: would it be better to add `const` keyword for clarity ?
```suggestion
	const annotation = "projectcontour.io/ingress.class"
```